NitroxClient/GameLogic/RemotePlayer.cs

```diff
@@ -385,10 +394,20 @@ public void UpdateEquipmentVisibility(List<TechType> equippedItems)
     }
 
     /// <summary>
-    /// Makes the RemotePlayer recognizable as an obstacle for buildings.
+    /// Makes the RemotePlayer recognizable as an obstacle for buildings, and as a target for creatures
     /// </summary>
     private void SetupBody()
     {
+        // set as a target for reapers
+        EcoTarget sharkEcoTarget = Body.AddComponent<EcoTarget>();
+        sharkEcoTarget.SetTargetType(EcoTargetType.Shark);
+
+        EcoTarget playerEcoTarget = Body.AddComponent<EcoTarget>();
+        playerEcoTarget.SetTargetType(PLAYER_ECO_TARGET_TYPE);
```

Comment:
I assume this sets a value that the reapers will go for. If two players have the same value does this cause us an issue? Will it only attack one person or just say the nearest one?

Reply:
I think it's more like a category, that the repear will look after. Rather than an ID

Reply:
Perfect sounds good to go then